tests/el/_fixtures.yaml

```diff
@@ -1,20 +1,20 @@
 language: el
 areas:
-  - name: Kitchen
+  - name: Κουζίνα
     id: kitchen
-  - name: Living Room
+  - name: Σαλόνι
     id: living_room
-  - name: Bedroom
+  - name: Υπνοδωμάτιο
     id: bedroom
-  - name: Garage
+  - name: Γκαράζ
     id: garage
 entities:
-  - name: Bedroom Lamp
+  - name: Φως υπνοδωματίου
```

Comment:
```suggestion
  - name: Φωτιστικό Υπνοδωματίου
```

Reply:
-There is no need for change here, as the text for training the model is not case-sensitive. Moreover, the correct greek grammar format is the originally suggested one.
-Regarding the "Φωτιστικό" suggestion, I think it would be more convenient for the users to use the originally suggested translation as it would be faster to express the complete voice command. Maybe we could suggest sth like:
           - name: (Φως | Φωτιστικό) υπνοδωματίου 
although I am a bit concerned about that. What is your opinion?